cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,50 @@
 /// <reference types='cypress' />
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
+import ProductInfoPageObject from '../support/pages/productInfo.pageObject';
+import CartPageObject from '../support/pages/cart.pageObject';
+import PlaceOrderPageObject from '../support/pages/placeOrder.pageObject';
+
+const { generateFakeUser } = require('../support/fakeUser');
+
+const fakeUser = generateFakeUser();
+
+const homeСatalogue = new HomeAndCataloguePageObject();
+const productInfo = new ProductInfoPageObject();
+const cart = new CartPageObject();
+const placeOrder = new PlaceOrderPageObject();
 
 describe('', () => {
   before(() => {
-
+    homeСatalogue.visit();
   });
 
-  it('', () => {
+  it('should add product to the Cart', () => {
+    homeСatalogue.clickOnCategory('Laptops');
+    homeСatalogue.clickOnProduct('Sony vaio i7');
+
+    productInfo.clickOnButton('Add to cart');
+
+    cy.on('window:alert', (alertText) => {
+      expect(alertText).to.contains('Product added');
+    });
+
```

Comment:
```suggestion
    cy.on('window:alert', (alertText) => {
      expect(alertText).to.contains('Product added');
    });
```
wrap this method into the page object method